lms/static/js/dashboard/legacy.js

```diff
@@ -116,11 +116,8 @@
              $('#failed-verification-banner').addClass('is-hidden');
          });
 
-         $('#upgrade-to-verified').click(function(event) {
-             var user = $(event.target).closest('.action-upgrade').data('user'),
```

Comment:
@dianakhuang: `user` and `course` did not work and are not used on some of the other 'edx.course.enrollment.upgrade.clicked' events.  I think we get this for free.